linux/aux-tools/startup-misc.sh

```diff
@@ -17,3 +17,32 @@ fi
 
 # Hide mounted devices from qubes-block list (at first udev run, only / is mounted)
 udevadm trigger --action=change --subsystem-match=block
+
+if [ -n "$(qvm-features dom0 suspend-s0ix)" ]; then
```

Comment:
Move this out of the `if`?  Right now if this fails the error is ignored.

Reply:
That's kinda intentional - missing feature should be treated as disabled.